zone/client.h

```diff
@@ -801,7 +801,25 @@ class Client : public Mob
 
 	void IncreaseSkill(int skill_id, int value = 1) { if (skill_id <= EQ::skills::HIGHEST_SKILL) { m_pp.skills[skill_id] += value; } }
 	void IncreaseLanguageSkill(uint8 language_id, uint8 increase = 1);
-	virtual uint16 GetSkill(EQ::skills::SkillType skill_id) const { if (skill_id <= EQ::skills::HIGHEST_SKILL) { return(itembonuses.skillmod[skill_id] > 0 ? (itembonuses.skillmodmax[skill_id] > 0 ? std::min(m_pp.skills[skill_id] + itembonuses.skillmodmax[skill_id], m_pp.skills[skill_id] * (100 + itembonuses.skillmod[skill_id]) / 100) : m_pp.skills[skill_id] * (100 + itembonuses.skillmod[skill_id]) / 100) : m_pp.skills[skill_id]); } return 0; }
+	virtual uint16 GetSkill(EQ::skills::SkillType skill_id) const
```

Comment:
We need to make this more readable, this is insane even when I auto formatted it.

Let's break things out into readable variables that describe what things do. We have nested ternaries and it's so hard to make sense of this.

We're at a point as well where this should just be a whole function in `.cpp` not stuff in the header